symfony/test/functional/cypress/fixtures/user.json

```diff
@@ -1,8 +1,8 @@
 {
     "admin": {
         "userName": "Admin",
-        "password": "Admin@123",
-        "fullName": "Samantha Jayasinghe"
+        "password": "admin@OHRM123",
+        "fullName": "Jacqueline White"
```

Comment:
{your-admin-pwd} goes here

Reply:
Created a separate file as admin.json and added admin credentials inside it.